scripts/ci-bootstrap.sh

```
@@ -0,0 +1,20 @@
+# wailsjs/ bindings that main.js imports.
+set -euo pipefail
+cd "$(dirname "$0")/.."
+
+WAILS_VERSION=v2.12.0
+
+mkdir -p cmd/hivegui/frontend/dist
+echo placeholder > cmd/hivegui/frontend/dist/.placeholder
+
+go install "github.com/wailsapp/wails/v2/cmd/wails@${WAILS_VERSION}"
+
+export PATH="$(go env GOPATH)/bin:$PATH"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)